docs/tutorials/analysis/3D/event_sampling.ipynb

```diff
@@ -77,6 +77,8 @@
     "from pathlib import Path\n",
```

Comment:
<p>You are using the same dataset than previously, so the dataset does not need to have a time axis ? maybe that's worth to mention it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gammapy/gammapy/pull/3507/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>